homeassistant/components/lightwave/__init__.py

```diff
@@ -50,4 +76,17 @@
             async_load_platform(hass, "switch", DOMAIN, switches, config)
         )
 
+    trv = config[DOMAIN][CONF_TRV]
+    trvs = trv[CONF_TRVS]
+    if trv:
+        proxy_ip = trv[CONF_PROXY_IP]
+        proxy_port = trv[CONF_PROXY_PORT]
+        hass.data[LIGHTWAVE_TRV_PROXY] = proxy_ip
+        hass.data[LIGHTWAVE_TRV_PROXY_PORT] = proxy_port
+        lwlink.set_trv_proxy(proxy_ip, proxy_port)
+
+        entities = [CLIMATE_DOMAIN, SENSOR_DOMAIN]
```

Comment:
These are not entities rather platforms.
```suggestion
        platforms = [CLIMATE_DOMAIN, SENSOR_DOMAIN]
```